package.json

```diff
@@ -92,5 +92,14 @@
     "css",
     "live reload",
     "sync"
-  ]
+  ],
+  "bugs": {
+    "url": "https://github.com/browsersync/browser-sync/issues"
+  },
+  "main": "index.js",
+  "directories": {
+    "example": "examples",
+    "test": "test"
+  },
+  "license": "Apache-2.0"
```

Comment:
Not needed. https://github.com/BrowserSync/browser-sync/blob/913b51ea5c0c7f3a481c229c00b8d1eecea63471/package.json#L10-L15